doc/reference/devices_gpu.md

```diff
@@ -51,6 +51,83 @@ Add a specific GPU from the host system as a `physical` GPU device to an instanc
 
     lxc config device add <instance_name> <device_name> gpu gputype=physical pci=<pci_address>
 
+Add a specific GPU from the host system as a `physical` GPU device to an instance using the [Container Device Interface](https://github.com/cncf-tags/container-device-interface) (CDI) notation.
+This applies to container only through the `id` configuration option to pass the fully-qualified CDI name:
+
+    lxc config device add <instance_name> <device_name> gpu gputype=physical id=<fq_CDI_name>
+
+Here are some examples of fully-qualified CDI names:
+
+- `nvidia.com/gpu=gpu0` : this instructs LXD to operate a discrete GPU (dGPU) passthrough of brand Nvidia with the first discovered GPU on your system. You can use the `nvidia-smi` tool on your host to know which identifier to use.
+- `nvidia.com/gpu=igpu0` : this instructs LXD to operate an integrated GPU (iGPU) passthrough of brand Nvidia with the first discovered iGPU on your system. An iGPU won't be detected by the `nvidia-smi` tool, so we encourage the user to use the `nvidia-ctk` tool through `nvidia-ctk cdi generate` to list the available iGPU devices.
+- `nvidia.com/gpu=all` : this instructs LXD to pass all the host GPUs of brand Nvidia through the container.
+
+Adding a device with the CDI notation is particularly useful if you have Nvidia runtime libraries and configuration installed on your host and that you want to pass these files to your container. Let's take the example of the iGPU passthrough:
+
+Your host is an Nvidia AGX / IGX board and therefore you have an Nvidia Tegra SoC with a powerful iGPU. You also have the Jetson SDK installed on the host, giving you plenty of useful libraries to handle AI workloads. You would want to create a LXD container and run an inference job inside the container using the iGPU as a backend. You would also like the inference job to be ran inside Docker container (or whatever OCI-compliant runtime). You could do something like this:
```

Comment:
This sounds very marketing-like. :wink: 
Is the part about Jetson actually required? Isn't what we need to say just that you have a powerful GPU that you want to use in a container?

Reply:
The part about Jetson is not stricly needed in the sense that the passthrough will work without it but if wou want to actually use the iGPU for something useful inside the container, then you need the libraries that come with the SDK.

Reply:
fixed